README.md

```diff
@@ -139,32 +102,90 @@ The **#matrix** project has some environment variables that important to define.
 
-## Running in Production
+|                                 Login                                 |                                   Login in Dark Mode                                    |
+| :-------------------------------------------------------------------: | :-------------------------------------------------------------------------------------: |
+| <img src="docs/img/matrix-login.png" title="Login page" width="100%"> | <img src="docs/img/matrix-dark-login.png" title="Login page in Dark Mode" width="100%"> |
+		
+### Docker Compose
+
+If you want run the **#matrix**, you need [docker-compose](https://docs.docker.com/compose/) and follow steps:
+
+1. Clone this repository `git clone git@github.com:ResultadosDigitais/matrix.git`
+
+2. We are using Google to authorizations, you need create a credential [here](/docs/GOOGLE-CREDENTIAL-STEP-BY-STEP.md) you can follow step by step
+
+3. duplicate file `variables.example.env` and rename to `variables.env`, after that set your environment variables;
+
+4. Run application with docker compose:
+
+		$ docker-compose up
+
+5. Open your browser and access: 
+
+		http://localhost:8080/
+
+6. When you finish, you can run:
+
+		$ docker-compose down
+
+### On GCP 
+
+If you prefer, you can run **#matrix** on GCP:
+
+[![Run on Google Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run?git_repo=https://github.com/ResultadosDigitais/matrix&revision=gcp-deploy-button)
+
+### On Heroku
+
+If you prefer, you can run **#matrix** in Heroku: 
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/ResultadosDigitais/matrix)
+
+### Production concerns
 
 If you will run in production we strongly recommend you close your environment using an internal VPN. In this solution everybody with the link and a valid google credential can enter your virtual office. Because this is important for you to maintain your environment closed. Or you can define a variable `WHITELIST_DOMAINS` to limit only authorized users to enter in the **#matrix**. You have to choose a strong key to the `COOKIE_SESSION_SECRET` and have to put in `GOOGLE_CALLBACK_URL` your production domain.
 
 		WHITELIST_DOMAINS=["@domain1.com","@domain2.com"]
 
+## Versions
+
+| Version | Name | Description | Docs |
+| --- | --- | --- | --- |
+| 2.0.0 | Seraph | New authentication | [Migration guide to 2.0.0](/docs/MIGRATION-TO-V2.md) |
+| 1.1.0 | Morpheus | New layout | [Pull request](https://github.com/ResultadosDigitais/matrix/pull/174) |
+| 1.0.0 | Neo | The one project | - |
+
 
 ## Contributing
 
 We encourage you to contribute to The **#matrix**!
 
 Everyone interacting in **#matrix** codebase, issue trackers, chat rooms, and mailing lists is expected to follow [code of conduct](docs/CODE_OF_CONDUCT.md).
 
+## Frequently Asked Questions
+
+Some questions come up over and over again. Check here first:
+[FAQ](docs/faq.md)
+
+## Get in Touch
```

Comment:
Here you don't give the title space, in the previous title you give it. Use the vscode markdown plugin and have the file formatted.